### DISCUSSION TITLE
How can I make tailwind and sass share the same color variables?

### DISCUSSION THREAD

**Discussion by RayPS** [tailwindlabs/tailwindcss#7611] [Help]
I have custom color variables defined in `_variable.scss` & `tailwind.config.js`, but whenever I want to change or add a new color I need to modify the both files.
Is there a way to let one of them inherit the colors from the other one?

`_variable.scss`
```scss
$color-primary: #28AA4B;
$color-danger: #FF7800;
```

`tailwind.config.js`
```javascript
module.exports = {
  content: [
    './index.html',
    './src/**/*.vue'
  ],
  theme: {
    extend: {
      colors: {
        primary: '#28AA4B',
        danger: '#FF7800'
      }
    }
  },
  plugins: []
}
```

I tried using `'theme(colors.primary)'` but it wont work with sass color modifiers like `opacify($color-primary, 0.25)`

**Comment by Serenit8** [comment]
I like the way Tailwind defines the colors and would like to use that outside of a Tailwind project.

For that I have defined two colors that I am already generating as classes (bg-|text-):

I would like to (if possible) generate the appropriate SCSS variables. e.g. $blue-300: #183f6d;

Is that possible?
[paymydoctor.com](https://www.paymydoctor.ltd/)
